### ISSUE TITLE
Canceling login started from the client triggers “Oops! Something went wrong” for the first login session

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#3315]
**FxA Train:**

* 229;

**VPN version:**

* 2.202204071128;

**Guardian version:**

* V1.23.b1 stage;

**Platforms:**

* All;

**Prerequisites:**

* Have the VPN app installed;
* Have an account with subscription for Mozilla VPN;

**Steps to reproduce:**

1. Open the VPN app;
1. Click “Get Started”;
1. On the web, leave the tab open, but do not finalize log in;
1. From the client, click “Cancel” on the “Waiting for subscription…” screen;
1. Click “Get Started” again;
1. On the web, go to first triggered session and finalize log in;

**Expected results:**

* Users can finalize log in. A success message is displayed.

**Actual results:**

* **“Oops! Something went wrong”** message is displayed. The client is blocked in the  “Waiting for subscription…” screen.

**Notes:**

* If, after step 6, we go to the second triggered web session and log in, we are blocked again.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-2079)
┆Reporter: Valentina Virlics

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Same result - a “Something went wrong“ error page is displayed, while doing different steps:

  * Open the VPN client and log in with an account without subscription;
  * Click the back browser arrow while in VPN landing page;
  * In FxA page change the email and use an account with subscription;
  * Finalize log in => “Oops! Something went wrong” page is displayed and the client remains in loading.

URL of the error page is:

https://stage-vpn.guardian.nonprod.cloudops.mozgcp.net/vpn/client/login/error?error=state_token_inequality

Sarah Bird Is this something Guardian related?

Attaching video.

!oops.gif|width=1403,height=842!

Thank you!

**Comment by data-sync-user** [comment]
➤ Sarah Bird commented:

I can’t remember why we marked this is low priority. But regardless, I may have fixed it in the 2.12 release. Can you re-verify?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

I think it was set as low priority because user had to use 2 tabs, leave one, continue to second, come back to first, etc.

Currently we cannot verify is this is still the case, as the authentication process happens inside the app, with 2.12.

We can close it.

**Comment by data-sync-user** [comment]
➤ Betty Fleming commented:

Sarah Bird can we get story points on this please?
